src/internal/Subscriber.ts

```diff
@@ -113,6 +113,39 @@ export class Subscriber<T> extends Subscription implements Observer<T> {
     }
   }
 
+  /**
+   * @ignore
+   * @internal
+   * Synchronous errors thrown within `subscribe` and caught by `_trySubscribe`
+   * can only be reported using a subscriber's `error` method if the subscriber
+   * is not stopped. This method traverses the `destination` chain of
+   * subscribers and uses the appropriate unhandled mechanism to report the
+   * error if a stopped subscriber is found.
+   */
+  reportError(err?: any): void {
```

Comment:
This should be private or protected. And ideally named with `_` prefix to discourage external use.

Reply:
Done. It's now private and has a leading `_`. However, its being private means that a cast to `any` is required in `_trySubscribe` and in the tests.